sigar_unix.go

```diff
@@ -4,11 +4,16 @@
 
 package gosigar
 
-import "syscall"
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
 
 func (self *FileSystemUsage) Get(path string) error {
-	stat := syscall.Statfs_t{}
-	err := syscall.Statfs(path, &stat)
+	stat := unix.Statfs_t{}
+	err := unix.Statfs(path, &stat)
```

Comment:
Any reason why this changed?

Based on the comment from the golang.org/x/sys/unix I would have thought the `os` package is preferred.

> The primary use of this package [golang.org/x/sys/unix] is inside other packages that provide a more portable interface to the system, such as "os", "time" and "net". Use those packages rather than this one if you can.

Reply:
I read that you should not use `syscall`:

>NOTE: This package is locked down. Code outside the standard Go repository should be migrated to use the corresponding package in the golang.org/x/sys repository. That is also where updates required by new systems or versions should be applied. Signal, Errno and SysProcAttr are not yet available in golang.org/x/sys and must still be referenced from the syscall package. See https://golang.org/s/go1.4-syscall for more information. 

I thought it could be an easy clean up. I would rather remove this change now, so we can merge it asap.